### PR TITLE
Add community banners

### PR DESCRIPTION
**Pull request by nicksellen**
Branch URL: https://add-community-banners.dev.karrot.world/#/welcome

## What does this PR do?

A concept for community banners. With Karrot Days in mind, but also usable for other things.

Most users won't see the community forum, and this allows us to make posts from the community forum show up as banner announcements in karrot.

Here's how it works:
- there is a category in the forum with two topics: [Community Announcements](https://community.karrot.world/c/karrot/community-announcements/29)
- anyone can read those, but only "Karrot Contributors" group can post there
- there is one topic for dev.karrot.world (+ local dev) and one for karrot.world
- the frontend loads the latest post and displays it as a banner
- when the user clicks _close_ it will go away, it stores the post id in local storage
- next time the page loads it'll only show a banner if it has a higher post id than the one in local storage

It looks like this:

Screenshot 2022-06-22 at 12-29-39 Karrot - Start a group become a community

(this is the post on karrot --> https://community.karrot.world/t/community-announcements-dev/930/7)

todo:
- [ ] decide whether to only show when logged in
- [ ] maybe don't show if there are other banners too? (or certain other ones?)

## Checklist

- [ ] added a test, or explain why one is not needed/possible...
- [ ] no unrelated changes
- [ ] asked someone for a code review
- [ ] joined [#karrot:matrix.org](https://matrix.to/#/#karrot:matrix.org)
- [ ] added an entry to CHANGELOG.md (description, pull request link, username(s))
- [ ] tried out on a mobile device (does everything work as expected on a smaller screen?)

### CODE REVIEW

**Comment by nicksellen** [comment]
Chatting with Nathalie today, she had a nice idea that we could have one forum topic, and the latest post shows as an announcement.

**Comment by tiltec** [comment]
I like the idea to use discourse a CMS for Karrot, at least for non-essential things :)
It's probably quite easy to make the topic id configurable for those who want to self-host Karrot in future (keeping in mind that it's one of the nlnet goals to make self-hosting easier!)

**Comment by nicksellen** [comment]
Ok, ready for a review! With branch deployment up at https://add-community-banners.dev.karrot.world/#/welcome

I decided to leave it showing on all pages, even logged out. Potentially is useful, and not a big change if we decide otherwise, can ask for feedback in next tuesday meeting (but could merge already if approved...).

File: src/communityFeed/api/communityFeed.js [hunk 1]
```diff
@@ -3,14 +3,17 @@ import axios from '@/base/api/axios'
...
+  },
   async getMeta () {
     return convert((await axios.get('/api/community-feed/')).data)
   },
   async markSeen () {
     return (await axios.post('/api/community-feed/mark_seen/')).data
   },
   async latestTopics () {
-    const data = (await axios.get('/community_proxy/c/karrot.json')).data
+    const data = (await axios.get('/community_proxy/c/karrot/7.json')).data
```
Comment (tiltec) [inline src/communityFeed/api/communityFeed.js:16]: Is this the Karrot category?

**Review by tiltec — APPROVED** [review]

File: src/communityFeed/api/communityFeed.js (hunk 1 above)
Comment (nicksellen) [reply src/communityFeed/api/communityFeed.js:16]: Yes, `'/community_proxy/c/karrot.json'` redirects to `/community_proxy/c/karrot/7.json` ... but with the full URL, and CORS blocks it in local dev.

**Comment by nicksellen** [comment]
Here is a downside! I reused the same blue for "new update", so it looks like this when there are both...

doublebanner

**Pull request merged**